daemon/logger/copier.go

```diff
@@ -41,11 +43,10 @@ func (c *Copier) Run() {
 
 func (c *Copier) copySrc(name string, src io.Reader) {
 	defer c.copyJobs.Done()
-	reader := bufio.NewReader(src)
+	reader := bufio.NewReaderSize(src, MaxBytesPerLine)
 
 	for {
-		line, err := reader.ReadBytes('\n')
-		line = bytes.TrimSuffix(line, []byte{'\n'})
+		line, _, err := reader.ReadLine()
 
 		// ReadBytes can return full or partial output even when it failed.
```

Comment:
Sorry for missing that. Removed the comment here and ` || len(line) > 0` since "ReadLine either returns a non-nil line or it returns an error, never both."